core/src/main/resources/lib/form/textarea.jelly

```diff
@@ -51,8 +51,10 @@ THE SOFTWARE.
       If @field is specified, this will be inferred automatically,
       which is the recommended approach.
     </st:attribute>
-    <st:attribute name="checkMethod">
-      If specified, the HTTP method to use for input field will be checked (via AJAX)
+    <st:attribute name="checkMethod" use="optional" type="String">
+      If specified, the HTTP method to use for input field will be checked (via AJAX).
```

Comment:
nit: If I understand correctly, then maybe something like the following would be a little clearer:

> Specify 'post' (must be lowercase) to change the HTTP method used for the AJAX requests to `@checkUrl` from a GET to a POST. If any other value is specified then requests will use GET.